packages/blade/src/components/FileUpload/FileUpload.web.tsx

```diff
@@ -1,4 +1,5 @@
 import { useState, useCallback, useMemo, useRef, forwardRef } from 'react';
+import { flushSync } from 'react-dom';
```

Comment:
Is flushSync available in React17? If not we won't be able to use it as it will fail on consumer end.

Reply:
Yes, flushSync was initially added in React 16. I was able to find the PR: https://github.com/facebook/react/pull/10225. However, I think the team didn’t document it.

I have tested this branch on React 17. Here is the sandbox link:
https://codesandbox.io/p/sandbox/blade-testing-template-ky8wfs

Reply:
Broken csb link ^

Reply:
@anuraghazra can you try it now? 
https://codesandbox.io/p/sandbox/blade-testing-template-ky8wfs

Reply:
Cool